db/schema.rb

```diff
@@ -274,8 +274,8 @@
     t.boolean "enrolled_in_rrh", default: false
     t.boolean "enrolled_in_psh", default: false
     t.boolean "enrolled_in_ph", default: false
-    t.boolean "majority_sheltered"
```

Comment:
Not sure why these move around, but, there don't seem to be any changes here.

Reply:
Yeah, that was just me running migrations.